packages/notebook-extension/src/index.ts

```diff
@@ -750,6 +751,13 @@ const lineColStatus: JupyterFrontEndPlugin<void> = {
   autoStart: true
 };
 
+const completerPlugin: JupyterFrontEndPlugin<void> = {
+  id: '@jupyterlab/notebook-extensions:completer',
```

Comment:
You should rename the settings schema `completer.json` (insteand of `notebooks.json`) to be linked with this plugin.